weave/trace/weave_client.py

```diff
@@ -2117,7 +2131,13 @@ def _flush(self) -> None:
         if self.future_executor_fastlane:
             self.future_executor_fastlane.flush()
         if self._server_is_flushable:
-            self.server.call_processor.stop_accepting_new_work_and_flush_queue()  # type: ignore
+            # We don't want to do an instance check here because it could
+            # be susceptible to shutdown race conditions. So we save a boolean
```

Comment:
When would that happen?

Reply:
thats a tim comment that I removed in the previous pr. not exactly sure.

Reply:
literally the symbol itself could be gone by the time it is needed here. it was erroring on me before